content/6_Advanced/LineContainer.problems.json

```diff
@@ -66,6 +80,66 @@
         "kind": "link",
         "url": "http://boi2011.ro/resurse/tasks/2circles-sol.pdf"
       }
+    },
+    {
+      "uniqueId": "3130",
```

Comment:
use the right format for unique ids

poj-ID

Reply:
done